VagrantProvision.sh

```diff
@@ -27,65 +28,101 @@ if ! dpkg -l | grep --quiet wamerican-insane; then
     sudo dpkg-reconfigure dictionaries-common
 fi
 
-# Install deps for Cucumber tests
-sudo apt-get install ruby ruby-dev xvfb
-sudo gem install mime-types -v 2.6.2
-sudo gem install capybara -v 2.5.0
-sudo gem install cucumber capybara-webkit selenium-webdriver rspec capybara-screenshot
```

Comment:
I think you're right, I can revert that back to the simpler set of statements.  The gem installs are very slow overall, which frustrated me as I was trying to get VagrantProvision.sh working properly.  Now that it is working properly, and the provisioning happens the first time only, it doesn't really matter that they are slow.